sale_order_revision/tests/test_sale_order_revision.py

```diff
@@ -43,35 +41,34 @@ def test_order_revision(self):
 
         # Check the previous revision of the sale order
         revision_1 = sale_order_1.current_revision_id
-        self.assertEqual(sale_order_1.state, 'cancel')
-        self.assertFalse(sale_order_1.active)
+        self.assertEqual(sale_order_1.state, "cancel")
+        # self.assertFalse(sale_order_1.active)
 
         # Check the current revision of the sale order
         self.assertEqual(revision_1.unrevisioned_name, sale_order_1.name)
-        self.assertEqual(revision_1.state, 'draft')
+        self.assertEqual(revision_1.state, "draft")
         self.assertTrue(revision_1.active)
         self.assertEqual(revision_1.old_revision_ids, sale_order_1)
         self.assertEqual(revision_1.revision_number, 1)
-        self.assertEqual(revision_1.name.endswith('-01'), True)
+        self.assertEqual(revision_1.name.endswith("-01"), True)
         self.assertEqual(revision_1.has_old_revisions, True)
 
         # Create a new revision of the Sale Order
         self._revision_sale_order(revision_1)
         revision_2 = revision_1.current_revision_id
 
         # Check the previous revision of the sale order
-        self.assertEqual(revision_1.state, 'cancel')
-        self.assertFalse(revision_1.active)
+        self.assertEqual(revision_1.state, "cancel")
+        # self.assertFalse(revision_1.active)
```

Comment:
Same as above

Reply:
I tested this on my local machine. It's working with the uncommented line as well. No change in code coverage.
In this case there is actually a change from line 52 revision_1.active = True to this line revision_1.active = False. So I think we should keep this.